packages/decoupled/package.json

```diff
@@ -23,40 +23,40 @@
     "decoupled": "./bin/decoupled"
   },
   "dependencies": {
-    "@types/dotenv": "^4.0.3",
-    "@types/express": "^4.11.1",
-    "@types/jest": "^22.2.3",
-    "@types/lodash": "^4.14.109",
-    "@types/winston": "^2.3.9",
-    "basic-auth": "^1.1.0",
-    "body-parser": "^1.18.2",
-    "caporal": "^0.10.0",
-    "chalk": "^2.4.1",
-    "cron": "^1.3.0",
+    "@types/dotenv": "^6.1.1",
+    "@types/express": "^4.17.1",
+    "@types/jest": "^24.0.18",
+    "@types/lodash": "^4.14.137",
```

Comment:
@gassius Do we need those tight version specs all around? Wouldn't it be sufficient to just manage versions at the minor-level? Specific versions are locked by yarn.lock anyways.